src/lib/provable/crypto/crypto.ts

```diff
@@ -12,11 +17,22 @@ const Crypto = {
   createCurve(params: Crypto.CurveParams): Crypto.Curve {
```

Comment:
That would require a bigger change, but I would have used a more explicit name to understand which form of curve and which coordinates we're using.

`createCurve` -> `createWeierstrassAffineCurve`
`createCurveTwisted` -> `createTwistedEdwardsAffineCurve`

The space (i.e. affine vs projective) could be omitted as most of the time, in circuit, we use the affine coordinates.

Reply:
I am a big fan of this refactor, I have been getting confused as well while working on this project. Let's open an issue.